docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -86,6 +86,67 @@
         font-weight: hb-theme-font-weight(semibold);
       }
     }
+
+    .hb-card--horizontal-date-stacked & {
+      flex-wrap: wrap;
+    }
+
+    // Target subcontent container of date stacked horizontal cards which feature icons in place of labels.
+    &--label-icon {
+      padding-top: 6px;
+      margin-bottom: 8px !important;
+
+      @include grid-media-min('sm') {
+        padding-top: 12px;
+      }
+    }
+
+    // style the date, time, location, and speaker
+    // within the date stacked horizontal UI pattern
+    &-detail {
+      padding-right: hb-calculate-rems(28px);
+      margin-bottom: hb-calculate-rems(16px);
+
+      .views-label {
+        @include visually-hidden;
+      }
+
+      .field-content {
+        display: block;
+        font-size: hb-calculate-rems(16px);
+        padding: 0 hb-calculate-rems(4px) 0 hb-calculate-rems(28px);
+        line-height: 127%; // keep icons from being cropped at smaller screen sizes
+      }
+
+      // only apply calendar icon when the label exists
+      .views-label-field-hs-event-date + .field-content {
```

Comment:
We should add new classes for these icons fields (like `.hb-card__date-icon`) instead of targeting the `.views-label-` classes.

Reply:
I chatted with Bryan on this to make sure I was on the right path forward. Targeting the Drupal `.vews-label-` classes is fine but they needed to be scoped inside of a class specifically for the stying of the icons.

I created a `hb-card__icon` base class that has modifiers for `date`, `location`, and `speaker`. By pulling out the icon classes out from being nested under `hb-card__subcontent-detail` we make them available for placement in other twig templates which use the `hb-card` classes. (For future use cases.)